### ISSUE TITLE
Yarn install error

### ISSUE THREAD

**Issue by artisin** [webpack/loader-utils#67]
Not a huge issue, but yarn throws the following error upon install: `loader-utils@1.0.0: The engine "node" is incompatible with this module. Expected version "^4.0.0".` therefore causing the install to fail. I'm more or less pulling this issue to let others know to mitigate this error you must use `yarn install --ignore-flags` until the node version is fixed/updated in the `package.json`.

**Comment by shjyh** [comment]
`--ignore-flags` seems not work
yarn version 0.20.3

**Comment by LaurensRietveld** [comment]
For me the `--ignore-flags` didn't work when doing a `yarn upgrade` (also v 0.20.3). Instead, I added `--ignore-engines`

**Comment by swernerx** [comment]
Yep, the issue might be that this in package.json is probably wrong:

```json
  "engines": {
    "node": "^4.0.0"
  },
```

Better use:

```json
  "engines": {
    "node": ">=4.0.0"
  },
```

**Comment by jhnns** [comment]
@swernerx Thanks for pointing that out. Should be fixed with `1.0.2`
